dashboards/bgpNeighbor/bgpNeighbor.json

```diff
@@ -0,0 +1,173 @@
+{
+  "name": "BGP Neighbor Summary",
+  "description": null,
+  "pages": [
+    {
+      "name": "BGP Neighbor Summary",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Devices with BGP OIDs ",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 8,
+            "height": 4
+          },
+          "visualization": {
+            "id": "viz.table"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT uniqueCount(Index) as Neighbors\n, uniqueCount(remote_as)\n, filter(uniqueCount(Index),where bgpPeerAdminStatus = 'start') as AdminEnabledNeighbors  \n, filter(uniqueCount(Index),where bgpPeerState = 'established') as EstablishedNeighbors\n, filter(uniqueCount(Index),where bgpPeerState != 'established' and bgpPeerState is not null and bgpPeerAdminStatus = 'start'  ) as NotEstablishedNeighbors\nfacet device_name, instrumentation.name \nwhere `mib-name` = 'BGP4-MIB' and tags.container_service = 'snmp'\n limit max"
```

Comment:
Just want to double-check, will this value be the same for _all_ users who install this dashboard?

```
`mib-name` = 'BGP4-MIB'
```

Reply:
Otherwise, this looks great! Nice job!

Reply:
Yep, mib-name will be consistent to anyone who has this set of data.